sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -215,3 +215,103 @@ def test_post_port_sfp_dom_info_to_db(self):
+                                                                              'nic_lane2_precursor2': '0.7',
+                                                                              'nic_lane2_maincursor': '0.7',
+                                                                              'nic_lane2_postcursor1': '0.7',
+                                                                              'nic_lane2_postcursor2': '0.7',
+                                                                              'tor_self_lane1_precursor1': '0.7',
+                                                                              'tor_self_lane1_precursor2': '0.7',
+                                                                              'tor_self_lane1_maincursor': '0.7',
+                                                                              'tor_self_lane1_postcursor1': '0.7',
+                                                                              'tor_self_lane1_postcursor2': '0.7',
+                                                                              'tor_self_lane2_precursor1': '0.7',
+                                                                              'tor_self_lane2_precursor2': '0.7',
+                                                                              'tor_self_lane2_maincursor': '0.7',
+                                                                              'tor_self_lane2_postcursor1': '0.7',
+                                                                              'tor_self_lane2_postcursor2': '0.7',
+                                                                              'tor_peer_lane1_precursor1': '0.7',
+                                                                              'tor_peer_lane1_precursor2': '0.7',
+                                                                              'tor_peer_lane1_maincursor': '0.7',
+                                                                              'tor_peer_lane1_postcursor1': '0.7',
+                                                                              'tor_peer_lane1_postcursor2': '0.7',
+                                                                              'tor_peer_lane2_precursor1': '0.7',
+                                                                              'tor_peer_lane2_precursor2': '0.7',
+                                                                              'tor_peer_lane2_maincursor': '0.7',
+                                                                              'tor_peer_lane2_postcursor1': '0.7',
+                                                                              'tor_peer_lane2_postcursor2': '0.7'}))
+    def test_post_port_mux_static_info_to_db(self):
+        logical_port_name = "Ethernet0"
+        mux_tbl = Table("state_db", "mux_static_info_tbl")
```

Comment:
Suggest making table names uppercase, as they would be in the real application. (e.g., `STATE_DB`, `MUX_CABLE_STATIC_INFO`)

Reply:
fixed